config/samples/porter-hello.yaml

```diff
@@ -9,5 +9,7 @@ spec:
   namespace: operator
   name: hello
   bundle:
-    repository: getporter/porter-hello
-    version: 0.1.1
+    repository: carolynvs/porter-hello-nonroot:v0.1.0
```

Comment:
Switch to a bundle that both takes a parameter and generates outputs to pick up on file permission problems with running as nonroot